python_modules/dagster/dagster/_core/pipes/context.py

```diff
@@ -275,6 +277,58 @@ class PipesSession:
     message_reader_params: PipesParams
     context: OpExecutionContext
 
+    @cached_property
+    def default_remote_invocation_tags(self) -> Dict[str, str]:
+        """Remote execution environments commonly have their own concepts of tags or labels.
```

Comment:
IMO this should follow docstring convention of starting with a one-liner that says what this actually is. Can we add a one-sentence paragraph at the top:

```Key-value pairs encoding metadata about the launching Dagster process.```

Reply:
ok done